CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+## 0.9.1 - 2015-07-03
+### Changed
+- The Broccoli build system was replaced by the Gulp build system
+- Complete replacement of the Backbone models with a Flux structure.
+  The components retrieve their data from stores via events on place.
+  Actions now fetch data from the Marathon API through an replaceable
+  Ajax-wrapper.
+
+### Fixed
+- Allow app creation with 0 instances
```

Comment:
@aldipower it would be cool if you could mention the ticket # here like [here](https://github.com/mesosphere/marathon/blob/master/changelog.md#fixed-bugs)